src/main/java/view/planning/EventPlanningWindow.java

```diff
@@ -1,201 +1,265 @@
-        JPanel summaryPanel = new JPanel();
-        summaryPanel.setLayout(new BoxLayout(summaryPanel, BoxLayout.Y_AXIS));
-        summaryPanel.add(currentDate);
-        summaryScrollPane = new JScrollPane(summaryText);
-        summaryPanel.add(summaryScrollPane);
-        return summaryPanel;
+    private JDateTimePicker createJDateTimePicker(){
+        Date date = new Date();
+        JDateTimePicker dateTimePicker = new JDateTimePicker();
+        gridBagConstraints = createGridBagConstraints();
+        gridBagConstraints.fill = GridBagConstraints.BOTH;
+        gridBagConstraints.weightx = 0.3;
+        gridBagConstraints.weighty = 0.15;
+        gridBagConstraints.gridx = 1;
+        gridBagConstraints.gridy = 0;
+        dateTimePicker.setFormats( DateFormat.getDateTimeInstance( DateFormat.SHORT, DateFormat.MEDIUM ) );
+        dateTimePicker.setTimeFormat( DateFormat.getTimeInstance( DateFormat.MEDIUM ) );
+        dateTimePicker.setDate(date);
+        return dateTimePicker;
     }
 
-    private JPanel createAddingPanel(JLabel personNameLabel, JTextField personNameTextField,
-                                     JLabel personSurnameLabel, JTextField personSurnameTextField,
-                                     JLabel thingNameLabel, JTextField thingNameTextField, JButton addingButton)
+    private JLabel createSummaryTitleJLabel() {
+        JLabel summaryTitleJLabel = new JLabel("Summary");
+        summaryTitleJLabel.setHorizontalAlignment(SwingConstants.CENTER);
+        gridBagConstraints = createGridBagConstraints();
+        gridBagConstraints.fill = GridBagConstraints.BOTH;
+        gridBagConstraints.weightx = 0.4;
+        gridBagConstraints.weighty = 0.15;
+        gridBagConstraints.gridx = 2;
+        gridBagConstraints.gridy = 0;
+        return summaryTitleJLabel;}
+
+    private JList createUserList()
     {
-        JPanel addingPanel = new JPanel();
-        addingPanel.setLayout(new BoxLayout(addingPanel, BoxLayout.Y_AXIS));
-        addingPanel.add(personNameLabel);
-        addingPanel.add(personNameTextField);
-        addingPanel.add(personSurnameLabel);
-        addingPanel.add(personSurnameTextField);
-        addingPanel.add(thingNameLabel);
-        addingPanel.add(thingNameTextField);
-        addingPanel.add(addingButton);
-        return addingPanel;
+        String defaultUsers[]= { "Macieg", "Mihau", "Baula", "Woocash", "Macieg", "Mihau", "Baula", "Woocash",
+                "Macieg", "Mihau", "Baula", "Woocash", "Macieg", "Mihau", "Baula", "Woocash"};
+        JList userList = new JList(defaultUsers);
+        userList.setSelectionMode(ListSelectionModel.SINGLE_INTERVAL_SELECTION);
+        userList.setLayoutOrientation(JList.VERTICAL);
+        userList.setVisibleRowCount(-1);
+        return userList;
+    }
+
+    private JScrollPane createScrollBarUserList(JList jlist) {
+        JScrollPane jScrollPane = new JScrollPane(jlist);
+        gridBagConstraints = createGridBagConstraints();
```

Comment:
Responsibilities are mixed here. The method does two things: create component, create layout constraints. There should be two methods instead